src/command_modules/azure-cli-resource/azure/cli/command_modules/resource/tests/test_resource.py

```diff
@@ -569,11 +569,11 @@ def test_show_built_in_policy(self):
 class ManagedAppDefinitionScenarioTest(ScenarioTest):
     @ResourceGroupPreparer()
     def test_managedappdef(self, resource_group):
-        location = 'westcentralus'
+        location = 'eastus2euap'
```

Comment:
Move this to be inside `@ResourceGroupPreparer()`, then expose a resource_group_location in the method signature. Do the same on the new test below